Editor/AGS.Types/CustomProperties.cs

```diff
@@ -15,6 +15,12 @@ public class CustomProperties : ICustomTypeDescriptor
 
         static public CustomPropertySchema Schema;
 
+        [AGSNoSerialize()]
+        public CustomPropertyAppliesTo AppliesTo
+        {
+            set { _appliesTo = value; }

```

Comment:
Please add a getter here too, it's strange to see a property that has only setter.

Reply:
Added in https://github.com/adventuregamestudio/ags/compare/2f81b98829f5c1d9e45ede5c740e4d7d07f7aec4..1a96c44d261da4dc6c5d884a9287f4690389e8b7 , but no one should never get it afaict.